[2024-08-30T17:56:34.717+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Dag_S3.extract_hr scheduled__2024-08-30T16:00:00+00:00 [queued]>
[2024-08-30T17:56:34.736+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Dag_S3.extract_hr scheduled__2024-08-30T16:00:00+00:00 [queued]>
[2024-08-30T17:56:34.738+0000] {taskinstance.py:2170} INFO - Starting attempt 7 of 12
[2024-08-30T17:56:34.779+0000] {taskinstance.py:2191} INFO - Executing <Task(SqlToS3Operator): extract_hr> on 2024-08-30 16:00:00+00:00
[2024-08-30T17:56:34.791+0000] {standard_task_runner.py:60} INFO - Started process 280 to run task
[2024-08-30T17:56:34.822+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ETL_Dag_S3', 'extract_hr', 'scheduled__2024-08-30T16:00:00+00:00', '--job-id', '127', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag_s3.py', '--cfg-path', '/tmp/tmpm98o977o']
[2024-08-30T17:56:34.853+0000] {standard_task_runner.py:88} INFO - Job 127: Subtask extract_hr
[2024-08-30T17:56:35.157+0000] {task_command.py:423} INFO - Running <TaskInstance: ETL_Dag_S3.extract_hr scheduled__2024-08-30T16:00:00+00:00 [running]> on host b5212bcfc9a5
[2024-08-30T17:56:35.831+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='jasontran' AIRFLOW_CTX_DAG_ID='ETL_Dag_S3' AIRFLOW_CTX_TASK_ID='extract_hr' AIRFLOW_CTX_EXECUTION_DATE='2024-08-30T16:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='7' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-30T16:00:00+00:00'
[2024-08-30T17:56:35.892+0000] {base.py:83} INFO - Using connection ID 'PostgreSQL_conn' for task execution.
[2024-08-30T17:56:35.945+0000] {base.py:83} INFO - Using connection ID 'PostgreSQL_conn' for task execution.
[2024-08-30T17:56:36.127+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/providers/common/sql/hooks/sql.py:221 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2024-08-30T17:56:36.235+0000] {sql_to_s3.py:179} INFO - Data from SQL obtained
[2024-08-30T17:56:36.240+0000] {sql_to_s3.py:186} INFO - Writing data to temp file
[2024-08-30T17:56:36.288+0000] {sql_to_s3.py:189} INFO - Uploading data to S3
[2024-08-30T17:56:36.316+0000] {base.py:83} INFO - Using connection ID 'AWS_S3_conn' for task execution.
[2024-08-30T17:56:36.318+0000] {connection_wrapper.py:381} INFO - AWS Connection (conn_id='AWS_S3_conn', conn_type='aws') credentials retrieved from extra.
[2024-08-30T17:56:38.574+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=ETL_Dag_S3, task_id=extract_hr, execution_date=20240830T160000, start_date=20240830T175634, end_date=20240830T175638
[2024-08-30T17:56:38.624+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-08-30T17:56:38.726+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
